### PR TITLE
Akiamara

### PR DESCRIPTION
**Pull request by tmagicturtle**

### CODE REVIEW

**Comment by ghost** [comment]
js, I would’ve done the ability differently. Also, there’s no innate ability, just to note.

**Comment by tmagicturtle** [comment]
I was implementing the Mold Breaker effect as an innate since the ability swapped stats and I thought it couldn't do 2 different things. The ability probably isn't done ideally - the server kept crashing when I tried to implement Heart Swap in an onStart or onSwitchIn event, and in the thread they mention having it "use Heart Swap". Obviously it probably wasn't supposed to be literal, but... for now, do not merge, I'll keep trying to get it working without calling a move.

File: data/mods/ssb/moves.js [hunk 1]
```diff
@@ -231,6 +231,35 @@ let BattleMovedex = {
...
+		},
+		onModifyMove(move, pokemon) {
+			if (pokemon.getStat('atk', false, true) > pokemon.getStat('spa', false, true)) move.category = 'Physical';
+		},
+		onPrepareHit(target, source) {
+			this.add('-anim', source, 'Focus Punch', target);
+		},
+		ignoreAbility: true,
+		ignoreEvasion: true,
+		ignoreDefensive: true,
```
Comment (HoeenCoder) [inline data/mods/ssb/moves.js:258]: The custom dosent do any of these?

File: data/mods/ssb/abilities.js [hunk 2]
```diff
@@ -98,6 +98,34 @@ let BattleAbilities = {
 			this.boost({atk: 1, def: -1, spd: -1, spe: 1});
 		},
 	},
+	// Akiamara
+	toxicswap: {
+		shortDesc: "On switch-in, this Pokemon swaps all stat changes with the opponent. Ignores abilities.",
```
Comment (HoeenCoder) [inline data/mods/ssb/abilities.js]: missing `desc`

File: data/mods/ssb/abilities.js [hunk 3]
```diff
@@ -98,6 +98,34 @@ let BattleAbilities = {
...
+			for (let i in target.boosts) {
+				// @ts-ignore
+				targetBoosts[i] = target.boosts[i];
+				// @ts-ignore
+				pokemonBoosts[i] = pokemon.boosts[i];
+			}
+			target.setBoost(pokemonBoosts);
+			pokemon.setBoost(targetBoosts);
+
+			this.add('-swapboost', pokemon, target, '[from] move: Heart Swap');
```
Comment (HoeenCoder) [inline data/mods/ssb/abilities.js]: Shouldn't be `[from] move: Heart Swap`, you can exclude it and use a custom this.add after this line for a custom message.

File: data/mods/ssb/moves.js (hunk 1 above)
Comment (tmagicturtle) [reply data/mods/ssb/moves.js:258]: It does. "Effects: Physical if user's Atk > Sp. Atk. Ignores Abilities and stat stage changes."

**Pull request merged**